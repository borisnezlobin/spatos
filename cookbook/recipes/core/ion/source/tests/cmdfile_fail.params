-c
echo 1
Hello
World
