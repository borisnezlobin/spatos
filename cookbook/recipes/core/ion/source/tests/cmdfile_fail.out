either execute command or file(s)
